### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by aydwi** [jothepro/doxygen-awesome-css#83] [General]
Excellent work on this stylesheet `jothepro`. Doxygen looks so fresh! And I agree, Doxygen's HTML documentation is very well structured and I'm very happy using it with your theme to document my new C++ library.

My sincere thanks to you for taking the time to make this stylesheet and sharing it with everyone.

**Comment by jothepro** [comment]
Thanks for you kind feedback! I am happy to hear that! 😊

**Reply by Rob-bb** [reply]
Yes, a big thank you from me as well :)

**Comment by Elphikaus** [comment]
WOW!!!!

So I have been working on a package and been using the stock standard Doxygen, like the premise of this repo -- it is dated. LIke 90's tile or something.

This repo is beautiful!!!

Excellent.

**Comment by randolf** [comment]
This addition to Doxygen is truly excellent -- it makes my documentation look professional and modern, and I feel that it's worthy of inclusion with the official Doxygen distributions so that more people can benefit from using it.
